triplea-game/triplea
# Remove system checks

## Overview
Instead of pro-actively alerting the user to a lack of network
connectivity, instead we allow that message only when the user performs
an action that would require network. EG: show error when user clicks
'play online' rather than at launch.

The temp file check is very unlikely to ever catch a problem, between
these two points there is not much value in the system check.

Beyond those points, the messaging and purpose of the system check error
message is confusing, instead of trying to fix it with the above
problems, will instead simply remove them.



## Functional Changes
'system check' functionality is removed. Instead we will show a user an error message when they try to connect to network resources and otherwise abandon the pro-active check we can create a temp file.

## Manual Testing Performed
-Launched game with network turned on/off to verify system check message no longer appeared.
